src/test/integration/integrationtests.py

```diff
@@ -33,31 +49,33 @@ def test_person_by_ids(self):
 
         # identifiers and return ids should matched
         person_data = osu_id_res.json()['data'][0]
-        ids_list = [
-            (person_data['id'], osu_id),
-            (person_data['attributes']['username'], onid),
-            (person_data['attributes']['osuUID'], osuuid)
-        ]
+        ids_list = [(person_data['id'],
+                     osu_id), (person_data['attributes']['username'], onid),
+                    (person_data['attributes']['osuUID'], osuuid)]
         for res_id, param_id in ids_list:
             self.assertEqual(res_id, param_id)
             self.assertIsInstance(res_id, str)
 
         # should return the same person if queried by each identifier
-        self.assertTrue(osu_id_res.content == onid_res.content == osuuid_res.content)
+        self.assertTrue(
+            osu_id_res.content == onid_res.content == osuuid_res.content)
 
     def test_phones(self):
         phones_res = utils.get_person_by_osu_id(phones_osu_id)
 
         home_phone = phones_res.json()['data']['attributes']['homePhone']
-        alternate_phone = phones_res.json()['data']['attributes']['alternatePhone']
+        alternate_phone = phones_res.json()['data']['attributes'][
+            'alternatePhone']
         primary_phone = phones_res.json()['data']['attributes']['primaryPhone']
         mobile_phone = phones_res.json()['data']['attributes']['mobilePhone']
```

Comment:
For this block, could you store `phones_res.json()` as a variable (or even `phones_res.json()['data']['attributes']`) so that we won't need to call `.json()` function every time and also could make the code neater?

Reply:
Sure, I was using google's yapf to format the code which is why these lines are changed.